spec/dummy/app/models/arm.rb

```diff
@@ -13,4 +13,11 @@ def social?
   def woz?
     has_woz
   end
-end
\ No newline at end of file
+
+  def bit_core_tools
+    self
+  end
+
+  def find_by_type(type)
```

Comment:
because the full stack feature specs run that method.  And instead of creating a `BitCore::Tool`class creating migration and associations, I think this is by far simpler.